File: Moviles/Ejercicio1/app/src/main/java/com/example/ejercicio1/MainActivity.java
package com.example.ejercicio1;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    int sumando1 = 0;
    int sumando2 = 0;

    TextView calculo = findViewById(R.id.calculo);

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        inicializarBotones();
    }

    public void inicializarBotones() {
        ArrayList<Button> listaBotones = new ArrayList<>();
        

    }
}

File: Moviles/WorkSpace Moviles/EjemRecycledView/app/src/main/java/com/example/ejemrecycledview/MainActivity.kt
package com.example.ejemrecycledview

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.GridLayout
import android.widget.LinearLayout
import androidx.recyclerview.widget.GridLayoutManager
import androidx.recyclerview.widget.LinearLayoutManager
import com.example.ejemrecycledview.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.rvList.layoutManager = LinearLayoutManager(this)

        binding.rvList.adapter = PersonaAdapter(ArrayList<Persona>())


    }
}